spec/features/editing_content_settings/history_mode_spec.rb

```diff
@@ -67,12 +65,12 @@ def then_i_enable_political_status
   end
 
   def and_i_see_the_timeline_entry
-    within("#document-history") do
-      expect(page).to have_content(I18n.t!("documents.history.entry_types.political_status_changed"))
-    end
+    click_on "Document history"
+    expect(page).to have_content(I18n.t!("documents.history.entry_types.political_status_changed"))
   end
 
   def when_i_click_to_backdate_the_content
+    visit document_path(@edition.document)
```

Comment:
Can we click the "Document summary" link rather than visit a url directly? - the latter is discouraged in feature tests

Reply:
We certainly can!